index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <title>My-Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div id="nav-container">
+                <div class="shapes">
+                    <img src="assets/shapes/image_geometry_1.jpg" alt="">   
+                </div>
+                <div class="ic_icon">
+                    <img src="assets/shapes/ic_menu.jpg" alt="">
+                </div>
+            </div>
+        </nav>
+        <div class="headline-section">
+            <div class="header-text">
+                <p>Charlie</p>
+                <p>Giorgania</p>
```

Comment:
- [ ] For good semantic tags and for best practices, kindly make sure the main title (e.g., "I'm Marianne ...") is a `h1` tag an not a `<p>` tag

*